packages/zowe-explorer/CHANGELOG.md

```diff
@@ -4,8 +4,6 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 
 ## TBD Release
```

Comment:
this should be ## `2.6.1`

Reply:
I believe the automation will replace this headers too, based on what happened in this commit:
- https://github.com/t1m0thyj/vscode-extension-for-zowe/commit/ea1344c083e096e1bde6a288224c16bc4c2910ff

Reply:
ok, I didn't realize it changed the CHANGELOG headers too.